gist_content.rb

```diff
@@ -0,0 +1,38 @@
+require 'pathname'
+
+class Received_values
+  attr_reader :data
+
+  def initialize(pathname, description, state)
+    @pathname = pathname
+    @description = description
+    @state = state
+  end
+
+  def path_existence
+    if Dir.exist?(@pathname)
```

Comment:
Maybe you should use the ternary operator for better organization. 
https://www.rubyguides.com/2019/10/ruby-ternary-operator/